pkg/email/lore/parse.go

```diff
@@ -20,9 +22,28 @@ type Thread struct {
 	Messages  []*email.Email
 }
 
+type Series struct {
+	Subject   string
+	MessageID string
+	Version   int
+	Corrupted string // If non-empty, contains a reason why the series better be ignored.
+	Patches   []Patch
+	// TODO: add Cover *email.Email?
```

Comment:
If this one is the short-term leaving TODO, please delete it.
If it is a long-term living one, please describe your concern better. There is a small chance somebody else will read it in the future.

Reply:
Good point, dropped the TODO.